### PR TITLE
Update _resources.md

### PR DESCRIPTION
**Pull request by HannaTrotsenko**

### CODE REVIEW

File: source/includes/_resources.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "put_url":"https://steward_host/files/1ksksiaö",
+  "put_url":"https://steward_host/files/1ksksiao",
```
Comment (zuker) [inline source/includes/_resources.md]: why `1ksksiao` i think it should be `mwhsntzy`

Comment (yarikos) [reply source/includes/_resources.md]: Why not just `djygurda`?

Comment (yarikos) [reply source/includes/_resources.md]: @anton-kotenko @ghaiklor  we need at least one more opition

Comment (ghaiklor) [reply source/includes/_resources.md]: `koroleva pezd` will be just fine

**Pull request merged**